src/cli/onefuzz/api.py

```diff
@@ -1731,6 +1731,34 @@ def post(
         return self._req_model("POST", responses.TemplateValidationResponse, data=req)
 
 
+class Migration(Endpoint):
```

Comment:
can we add the new command under the `notifications` subcommand. 
I think it will make it easier to discover